Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AtomRenderPlugin.cpp

```diff
@@ -189,10 +198,21 @@ namespace EMStudio
         m_scaleManipulators.ConfigureView(
             AzToolsFramework::LinearManipulatorAxisLength(), AzFramework::ViewportColors::XAxisColor,
             AzFramework::ViewportColors::YAxisColor, AzFramework::ViewportColors::ZAxisColor);
+        m_scaleManipulators.InstallAxisLeftMouseDownCallback(
+            [this]([[maybe_unused]] const AzToolsFramework::LinearManipulator::Action& action)
+            {
+                const AZ::EntityId entityId = m_animViewportWidget->GetAnimViewportRenderer()->GetEntityId();
+                AZ::TransformBus::EventResult(m_mouseDownStartTransform, entityId, &AZ::TransformBus::Events::GetLocalTM);
+            });
         m_scaleManipulators.InstallAxisMouseMoveCallback(
             [this](const AzToolsFramework::LinearManipulator::Action& action)
             {
-                OnManipulatorScaled(action.LocalScale(), action.LocalScaleOffset());
+                // Since we are compulting a uniform scale, the delta scale should be the none-zero value from one of the three axis.
```

Comment:
Since we are computing a uniform scale, the delta scale should be a non-zero value from one of the three axes.

Reply:
since we are adding the delta scale, not multiply it, I don't think there is need to assert here.